app/models/__tests__/dturns.js

```diff
@@ -0,0 +1,132 @@
+// https://github.com/EnCiv/civil-pursuit/issues/196
+
+const { Mongo, Collection } = require('@enciv/mongo-collections')
+const Dturns = require('../dturns')
+const { beforeEach } = require('@jest/globals')
+
+const userId = '6667d5a33da5d19ddc304a6b'
+
+beforeEach(async () => {
+  await Dturns.deleteMany({})
+})
+
+beforeAll(async () => {
+  await Mongo.connect(global.__MONGO_URI__, { useUnifiedTopology: true })
+  await Dturns.setCollectionProps()
+})
+
+afterAll(async () => {
+  await Mongo.disconnect()
+})
+
+describe('Dturns Model', () => {
+  it('should be set up correctly', () => {
+    expect(Dturns.collectionName).toEqual('dturns')
+  })
+
+  it('should insert a valid document', async () => {
+    const validDoc = {
+      discussionId: 'discussionId',
+      userId: userId,
+      round: 0,
+      data: { user: 'userId', points: { point1: 'point1', point2: 'point2' } },
+    }
+    const result = await Dturns.insertOne(validDoc)
+    expect(result.acknowledged).toBe(true)
+  })
+
+  it('should not insert an invalid document', async () => {
+    const invalidDoc = {
+      data: 'not real data',
+    }
+    await expect(Dturns.insertOne(invalidDoc)).rejects.toThrow('Document failed validation')
+  })
+
+  it('should upsert when the upsert function is called', async () => {
+    const doc = {
+      userId: userId,
+      discussionId: '1',
+      round: 0,
+      shownStatementIds: { points: ['id1', 'id2'] },
```

Comment:
please use hex strings for id1 and id2, like '111', or paste in valid objectIds strings.